bochs/cpu/vmx.cc

```diff
@@ -2122,10 +2133,19 @@ Bit32u BX_CPU_C::VMenterLoadCheckGuestState(Bit64u *qualification)
   BX_CPU_THIS_PTR msr.sysenter_eip_msr = guest.sysenter_eip_msr;
   BX_CPU_THIS_PTR msr.sysenter_cs_msr  = guest.sysenter_cs_msr;
 
+#if BX_SUPPORT_UINTR
+  if (BX_CPUID_SUPPORT_ISA_EXTENSION(BX_ISA_UINTR)) {
+    BX_CPU_THIS_PTR uintr.uinv = guest.uintr_uinv;
+  }
+#endif
+
 #if BX_SUPPORT_VMX >= 2
   if (vmentry_ctrls & VMX_VMENTRY_CTRL1_LOAD_PAT_MSR) {
     BX_CPU_THIS_PTR msr.pat = guest.pat_msr;
   }
+#endif
+
+#if BX_SUPPORT_VMX >= 2
```

Comment:
Two adjacent blocks guarded by the same ``#if`` condition look suspicious.

Reply:
But they related to disjoint feature therefore I split them in purpose